packages/api/src/auth/accessToken.ts

```diff
@@ -0,0 +1,24 @@
+import type { APIGatewayProxyEvent } from 'aws-lambda'
+
+export type AccessToken = null | string
+
+/**
+ * This function returns the Authorization Token from the client request event headers
+ * to be decoded by decodeAuthToken() or otherwise used in token-based authentication
+ * when required to allow an application to access an API; hence: "Access Token".
+ *
+ * @returns `null` when the authorization token wasn't available in the event headers.
+ * @returns `string` otherwise.
+ */
+export const accessToken = async (event: {
```

Comment:
> getAuthorization

Right ... since that is the header name ... and even though people debate `authentication` (who you are) vs `authorization` (what you can do) in this case authorization does fit because the token would contain those permissions as well.

> That way we can support the Basic schema in the future.

Right.